Lib/test/test_unittest/test_program.py

```diff
@@ -71,15 +71,22 @@ def testExpectedFailure(self):
         def testUnexpectedSuccess(self):
             pass
 
-    class FooBarLoader(unittest.TestLoader):
-        """Test loader that returns a suite containing FooBar."""
+    class Empty(unittest.TestCase):
+        pass
+
+    class TestLoader(unittest.TestLoader):
+        """Test loader that returns a suite containing testsuite."""
```

Comment:
?

```suggestion
        """Test loader that returns a suite containing self.testcase."""
```

Reply:
That was a typo of the class initialization argument name, thanks